assets/css/contact.min.css

```diff
@@ -0,0 +1 @@
+.contact-form{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;display:inline-block;padding:30px 130px;width:100%;max-width:1e3px}.user-details .user-name{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;display:inline-block;width:50%;padding:0 2.5px 0 0}.user-details .user-email{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;display:inline-block;width:50%;padding:0 0 0 2.5px}.user-details input{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;display:block;box-shadow:inset 0 0 0 1px #e1e1e1;background:#f5f5f5;width:100%;padding:.85em 1em;border-radius:.25em;border:0;line-height:1.25em}.user-details input:focus{outline:0;box-shadow:inset 0 0 0 1px #afd9e0;background:#fff}.message{margin:5px 0}.message textarea{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;min-height:16em;min-width:100%;max-width:100%;border-radius:.25em;border:0;box-shadow:inset 0 0 0 1px #e1e1e1;background:#f5f5f5;padding:.85em 1em;font-family:inherit}.message textarea:focus{outline:0;box-shadow:inset 0 0 0 1px #afd9e0;background:#fff}.actions{margin-top:20px}.actions .g-recaptcha{display:inline-block;margin-bottom:21px}.actions ul{padding:0;margin:0}.actions li{display:inline-block;margin:0 20px}.actions input{position:relative;background:#3d3d3d;padding:.85em 3em;border-radius:.25em;cursor:pointer;border:0;color:#fff;font-size:13px}.actions input:hover{background:#4f4f4f}.actions input.reset-button{color:inherit;box-shadow:inset 0 0 0 1px #e1e1e1;background:0 0}.actions input.reset-button:hover{background:#fafafa}@media screen and (max-width:736px){.contact-form{padding:30px 16px}.user-details .user-name{display:block;width:100%;padding:0 0 2.5px}.user-details .user-email{display:block;width:100%;padding:2.5px 0 0}.message textarea{min-height:16em}.actions ul{text-align:center;padding:0;margin:0}.actions li{margin:0 10px}.actions li:nth-child(odd){margin-left:0}.actions li:nth-child(even){margin-right:0}.actions input{font-size:12px;padding:.85em 2.5em}}
```

Comment:
Exclude generated files from the commit, then make sure that the gitignore entry works (meaning, they aren't picked up again).